test/buffer_writer.spec.ts

```diff
@@ -0,0 +1,231 @@
+import * as assert from 'assert';
+import { describe, it } from 'mocha';
+import { BufferWriter } from '../src/buffer_writer';
+
+const varuint = require('varuint-bitcoin');
+
+describe('BufferWriter', () => {
+  function concatToBuffer(values: number[][]): Buffer {
+    return Buffer.concat(values.map(data => Buffer.from(data)));
+  }
```

Comment:
this is basically a workaround to `prettier` removing my formatting of `expectedBuffer` as groups